Gemfile

```diff
@@ -39,6 +39,7 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 gem "american_date"
 gem "default_value_for",              "~>3.0.1"
 gem "thin",                           "~>1.3.1"  # Used by rails server through rack
```

Comment:
i can try that :) the ticket only mentions switching the UI worker - was there a reason for that?